sections/main-article.liquid

```diff
@@ -35,7 +35,7 @@
           <header class="page-width page-width--narrow" {{ block.shopify_attributes }}>
             <h1 class="article-template__title" itemprop="headline">{{ article.title | escape }}</h1>
             {%- if block.settings.blog_show_date -%}
-              <span class="circle-divider caption-with-letter-spacing" itemprop="dateCreated pubdate datePublished">{{ article.published_at | time_tag: format: 'month_year' }}</span>
+              <span class="circle-divider caption-with-letter-spacing" itemprop="dateCreated pubdate datePublished">{{ article.published_at | time_tag: format: 'month_day_year' }}</span>
```

Comment:
I don't think we actually need to use a custom format but could leverage this instead: [time tag with shorthand format](https://shopify.dev/api/liquid/filters/html-filters#format). 
So it would end up being: `{{ article.published_at | time_tag: format: 'date' }}`